napalm_eos/eos.py

```diff
@@ -795,9 +795,9 @@ def get_arp_table(self):
 
         return arp_table
 
-    def get_ntp_peers(self):
+    def get_ntp_stats(self):
 
-        ntp_peers = dict()
+        ntp_stats = list()
```

Comment:
Is this you, @dbarrosop? :)

Indeed seems to make a difference between them, interesting. But on my machines the difference is not that high. If you want I can make the changes in the PRs submitted yesterday but I don't really think it will really make a difference in our case (using SSH, waiting for device to reply (and the priority is anyway very low for these processes), parsing the output etc.)

```python
0.025195837020874023
>>> timeit("list()")
0.07649493217468262
>>> timeit("list()")
```